openmc/settings.py

```diff
@@ -101,6 +101,13 @@ class Settings(object):
         Coordinates of the lower-left point of the Shannon entropy mesh
     entropy_upper_right : tuple or list
         Coordinates of the upper-right point of the Shannon entropy mesh
+    tabular_legendre : dict
```

Comment:
What do you think about just having this be a bool and then having another attribute called perhaps `num_tabular_points` that takes on the value of `num_points`?